docs/manual/java/guide/services/ServiceClients.md

```diff
@@ -59,11 +59,13 @@ All service calls with Lagom service clients are by default using circuit breake
 
 In the above example the default identifier is used for the `sayHi` method, since no specific identifier is given. The default identifier is the same as the service name, i.e. `"hello"` in this example. The `hiAgain` method will use another circuit breaker instance, since `"hello2"` is specified as circuit breaker identifier.
 
+### Circuit Breaker Configuration
+
 On the client side you can configure the circuit breakers. The default configuration is:
 
 @[circuit-breaker-default](../../../../../service/core/client/src/main/resources/reference.conf)
 
-That configuration will be used if you don't define any configuration yourself.
+That configuration will be used if you don't define any configuration yourself. You may want to configure the circuit breaker for a given endpointto ignore a certain exceptions. Lagom's client [[maps all 4xx and 5xx responses to Exceptions|ServiceErrorHandling]] and Lagom's Circuit Breaker defaults to considering all Exceptions as failures.  This is particularly useful when connecting to services where 4xx HTTP status codes are used to model business valid cases. For example, it may be a non-failure case to respond a 404 to a particular request. In that case the client will still receive a `NotFound` exception as a result of invoking the service all but you could add `"com.lightbend.lagom.javadsl.api.transport.NotFound"` to the circuit breaker whitelist so that it is not considered by the circuit breaker failure count.
```

Comment:
type "endpointto"

Reply:
"..invoking the service all but you could.." shouldn't the "all" be replaced by a comma>